### PR TITLE
Fix custom meta tags key error

### PR DESCRIPTION
**Pull request by MrNossiom**
This pull request fixes bug explained in (#666)

Closes: #666

### CODE REVIEW

**Comment by MrNossiom** [comment]
Hey @rodzy, here is my pull request for #666

**Review by rodzy — COMMENTED** [review]
This approach looks good to me 💭.

Just one question can this behavior be tested on the `buildTags.spec.tsx`? Did you try to replicate the issue with this solved?

Let's see what @garmeeh thinks about it, he needs to Approve this to be merge.

**Comment by garmeeh** [comment]
Hey @MrNossiom thanks for the contribution and thanks @rodzy for the insight on it 😄 

There is currently this #660 PR open that will have the same issue around keys. 

I like the solution but I don't think it would work as the keys are used to clean up tags out of the head on new pages if not needed, so by generating a random id each time this could lead to some issues. 

I think the best approach would be to add a prop `keyOverride` which will applications to manage the key that need to. That way if they wish to use a random id they can do so. 

Do you think the `keyOverride` approach would work for your use case?

**Comment by MrNossiom** [comment]
I found out that I was using additionalMetaTags for what was supposed to be links... 👎🏻

I don't think that there is other type of meta tags than these:
```ts
export type MetaTag = HTML5MetaTag | RDFaMetaTag | HTTPEquivMetaTag;
```

The idea of keyOverride in case of weird meta tags is a great idea!

**Comment by MrNossiom** [comment]
Hey @garmeeh,

This pull request is ready to merge.
Should I rebase my branch on beta and switch the PR destination to beta ?

Thanks

File: src/meta/buildTags.tsx [hunk 1]
```diff
@@ -653,7 +652,12 @@ const buildTags = (config: BuildTagsParams) => {
   if (config.additionalMetaTags && config.additionalMetaTags.length > 0) {
     config.additionalMetaTags.forEach(tag => {
       tagsToRender.push(
-        <meta key={tag.name || tag.property || tag.httpEquiv} {...tag} />,
+        <meta
+          key={`meta:${
```
Comment (MrNossiom) [inline src/meta/buildTags.tsx:656]: I don't know if we put a separator after the name for easier and cleaner debugging.

**Review by garmeeh — APPROVED** [review]
This looks good to me @MrNossiom. Thanks for the update.

**Comment by garmeeh** [comment]
@all-contributors please add @MrNossiom code doc

**Pull request merged**